.gitlab-ci.yml

```diff
@@ -35,63 +36,40 @@ before_script:
   - scripts/fetch_secret.sh gcp-credentials.json > gcp-credentials.json
   - export GOOGLE_APPLICATION_CREDENTIALS=$(pwd -P)/gcp-credentials.json
 
-unit_tests:
+.tests:
   stage: test
-  script:
-    - whoami
-    - ls -alh /
-    - make -j4 parallel_test
   except:
     - tags
     - schedules
 
+unit_tests:
+  extends: .tests
+  script:
+    - make -j4 parallel_test
+
 test_search:
-  stage: test
+  extends: .tests
   script:
     - make -j1 tests/test_search.py
-  except:
-    - tags
-    - schedules
 
 test_aws_indexer:
-  stage: test
+  extends: .tests
   variables:
     DSS_UNITTEST_OPTS: "-v TestAWSIndexer"
   script:
     - make -j1 tests/test_indexer.py
-  except:
-    - tags
-    - schedules
 
 test_gcp_indexer:
-  stage: test
+  extends: .tests
   variables:
     DSS_UNITTEST_OPTS: "-v TestGCPIndexer"
   script:
     - make -j1 tests/test_indexer.py
-  except:
-    - tags
-    - schedules
 
 test_subscriptions:
-  stage: test
+  extends: .tests
   script:
     - make -j1 tests/test_subscriptions.py
-  except:
-    - tags
-    - schedules
-
-test_sync_large_blob:
-  stage: test
-  variables:
-    DSS_TEST_MODE: "integration"
-  script:
-    - python tests/test_sync.py TestSyncDaemon.test_sync_large_blob
-  only:
-    refs:
-      - schedules
-    variables:
-      - $DSS_TEST_LARGE_FILE_SYNC
 
```

Comment:
test_sync_large_blob was removed, intentional? Otherwise looks much cleaner. 👍

Reply:
The large blob sync test will be performed under the new `scale_and_performance` stage.